content/resources.md

```diff
@@ -257,7 +285,80 @@ Below is the JSON standard format representing theses attribute options.
 }
 ```
 
-::: panel-link Want more details about the attribute option resource? [Check its endpoints here!](/api-reference.html#Attribute options)
+::: panel-link Want more details about the attribute option resource? [Check its endpoints here!](/api-reference.html#Attributeoptions)
+:::
+
+## Attribute group (2.0 only)
+
+To facilitate the work of Julia inside the PIM, we gather attributes into groups. These groups are called `Attribute groups`.
+
+Below is the JSON standard format representing these attribute groups.
+
+```json
+{
+  "code":"technical",
+  "sort_order":0,
+  "attributes": ["weight","width","height"],
+  "labels":{
+     "en_US":"Technical",
+     "fr_FR":"Informations techniques"
+  }
+}
+```
+```json
+{
+  "code":"marketing",
+  "sort_order":1,
+  "attributes": ["marketing_name","description"],
+  "labels":{
+     "en_US":"Marketing",
+     "fr_FR":"Marketing"
+  }
+}
+```
+
+::: panel-link Want more details about the attribute group resource? [Check its endpoints here!](/api-reference.html#Attributegroups)
+:::
+
+## Currency (2.0 only)
+If you want to store price information inside your PIM, you will need currencies.
```

Comment:
There is a blank line between the title and the text for the other description (attributes, etc). If it does not change anything to the generated text, forget this comment.

Reply:
:)